tests/integration/cattletest/core/test_allocation.py

```diff
@@ -25,40 +25,41 @@ def test_compute_free(admin_client, sim_context):
 
     containers = wait_all_success(admin_client, containers)
 
-    host = admin_client.reload(host)
+    host = internal_test_client.reload(host)
     assert host.computeFree == start_free
 
 
-def test_inactive_agent(admin_client, sim_context):
-    agent = create_type_by_uuid(admin_client, 'agent', 'inactive_test',
+def test_inactive_agent(internal_test_client, sim_context):
+    agent = create_type_by_uuid(internal_test_client, 'agent', 'inactive_test',
                                 uri='sim://inactive_test')
 
-    agent = wait_success(admin_client, agent)
+    agent = wait_success(internal_test_client, agent)
     host = agent.hosts()[0]
 
-    c = admin_client.create_container(imageUuid=sim_context['imageUuid'],
-                                      requestedHostId=host.id)
+    c = internal_test_client.create_container(
+        imageUuid=sim_context['imageUuid'], requestedHostId=host.id)
 
-    c = wait_success(admin_client, c)
+    c = wait_success(internal_test_client, c)
     assert c.state == 'running'
 
-    agent = wait_success(admin_client, agent.deactivate())
+    agent = wait_success(internal_test_client, agent.deactivate())
     assert agent.state == 'inactive'
 
-    c = admin_client.create_container(imageUuid=sim_context['imageUuid'],
-                                      requestedHostId=host.id)
+    c = internal_test_client.create_container(
+        imageUuid=sim_context['imageUuid'], requestedHostId=host.id)
 
-    c = wait_transitioning(admin_client, c)
+    c = wait_transitioning(internal_test_client, c)
     assert c.transitioning == 'error'
     assert c.transitioningMessage == 'Failed to find a placement'
     assert c.state == 'removed'
 
 
-def test_spread(admin_client, sim_context, sim_context2, sim_context3):
+def test_spread(admin_client, internal_test_client,
```

Comment:
This makes sense. Fixed this. Just curious, when you talk about complexity here, is that just about having an extra parameter in there to perform create container task which can also be done by internal_test_client?